pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -139,7 +139,7 @@ func isPendingPastDeadline(csr *capi.CertificateSigningRequest) bool {
 	// If there are no Conditions on the status, the CSR will appear via
 	// `kubectl` as `Pending`.
 	if len(csr.Status.Conditions) == 0 && isOlderThan(csr.CreationTimestamp, pendingExpiration) {
-		klog.Infof("Cleaning CSR %q as it is more than %v old and unhandled.", csr.Name, pendingExpiration)
+		klog.InfoS("Cleaning CSR as it is past expiration and unhandled.", "csr", klog.KObj(csr), "pendingExpiration", pendingExpiration)
```

Comment:
Name of keys should not be used to convey information what kind of expiration it is. 
```suggestion
		klog.InfoS("Cleaning CSR as it is past pending expiration and unhandled.", "csr", klog.KObj(csr), "age", time.Since(csr.CreationTime), "expiration": pendingExpiration)
```

Reply:
@serathius  I changed "pendingExpiration" to "expiration" but I didn't use `"age", time.Since(csr.CreationTime),` since csr doesn't have a property `CreationTime`. I also didn't use `"condition", "Pending"`, not sure what you were trying to do with this one?